server/sonar-server/src/test/java/org/sonar/server/issue/index/IssueIndexTest.java

```diff
@@ -630,6 +636,7 @@ public void filter_by_created_after_must_not_be_in_future() {
     }
   }
 
+  @Ignore
   @Test
   public void filter_by_created_at() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)